geo-types/src/coordinate.rs

```diff
@@ -232,21 +293,26 @@ use num_traits::Zero;
 /// assert_eq!(p.x, 0.);
 /// assert_eq!(p.y, 0.);
 /// ```
-impl<T: CoordNum> Coordinate<T> {
+impl<T: CoordNum, Z: CoordNum, M: CoordNum> Coordinate<T, Z, M> {
+    #[inline]
```

Comment:
I'm concerned that adding in all these inlines might cover up unrelated perf regressions. 

Can you separate the unrelated inlines into a separate PR?